util/include/util/threadpool.h

```diff
@@ -4,6 +4,7 @@
 #include <atomic>
 #include <chrono>
 #include <condition_variable>
+#include <functional>
```

Comment:
The default for a template parameter is `std::function`. This include was needed ever since #120, but the files this header was included to already made `<functional>` available.